astroquery/query.py

```diff
@@ -4,12 +4,13 @@
 import abc
 import pickle
 import hashlib
+import io
 import os
 import requests
 
 from astropy.extern import six
 from astropy.config import paths
-from astropy import log
+from astropy.logger import log
```

Comment:
Note: Does not change functionality. Just want to be consistent with other modules in this package.